cassovary-core/src/test/scala/com/twitter/cassovary/graph/bipartite/BipartiteGraphSpec.scala

```diff
@@ -90,7 +89,7 @@ class BipartiteGraphSpec extends Specification {
     graph = new BipartiteGraph(leftSide, rightSide, BipartiteGraphDir.LeftToRight)
```

Comment:
no need to set a graph variable. just have the last line be 
new BipartiteGraph(...